include/zephyr/bluetooth/hci.h

```diff
@@ -1899,6 +1899,21 @@ struct bt_hci_rp_le_read_iso_tx_sync {
 	uint8_t  offset[3];
 } __packed;
 
+#define BT_HCI_CIG_ID_MAX                       0xFE
+#define BT_HCI_CIG_CIS_COUNT_MAX                0x1F
+#define BT_HCI_CIG_SDU_INTERVAL_MIN             0x0000FF
+#define BT_HCI_CIG_SDU_INTERVAL_MAX             0x0FFFFF
+#define BT_HCI_CIG_WORST_CASE_SCA_VALID_MASK    0x07
+#define BT_HCI_CIG_PACKING_VALID_MASK           0x01
+#define BT_HCI_CIG_FRAMING_VALID_MASK           0x01
+#define BT_HCI_CIG_MAX_TRANSPORT_LATENCY_MIN    0x0005
+#define BT_HCI_CIG_MAX_TRANSPORT_LATENCY_MAX    0x0FA0
+#define BT_HCI_CIG_CIS_ID_VALID_MAX             0xEF
+#define BT_HCI_CIG_MAX_SDU_VALID_MASK           0x0FFF
+#define BT_HCI_CIG_PHY_VALID_MASK               0x07
+#define BT_HCI_CIG_ISO_INTERVAL_MIN             0x0004
+#define BT_HCI_CIG_ISO_INTERVAL_MAX             0x0C80
```

Comment:
Just a note: If we make all these CIG specific, then they will be of no use to the application as @cvinayak suggested, as the ISO API does not differ between CIS and BIS for the majority of these values. 

@cvinayak is that OK for you, or should we make the values that are common between unicast and broadcast use a common generic prefix like `BT_HCI_ISO` instead of `BT_HCI_CIG`?

Reply:
Perhaps we can change these to `_ISO_` when BIS needs them?